tests/codecs/test_base64.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from mlserver.codecs import Base64Codec
+from mlserver.types import RequestInput, ResponseOutput
+
+
+@pytest.mark.parametrize(
+    "decoded, expected",
+    [
+        (
+            [b"Python is fun"],
+            ResponseOutput(
+                name="foo",
+                shape=[1, 20],
+                datatype="BYTES",
+                data=b"UHl0aG9uIGlzIGZ1bg==",
+            ),
+        ),
+        (
+            ["Python is fun"],
```

Comment:
same as about test case?

Reply:
There should be some small variations on each test case. E.g. one uses a "binary string" whereas the other uses a regular one, or one returns multiple elements, etc.

Reply:
oh yes! should we add a note perhaps or run the test twice on each usecase for example if the testcase is "Python is fun" -> we run once with [b"Python is fun"] and with ["Python is fun"] to reduce redundancy?